index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tasnimul hasan portfolio</title>
+    <link href="style.css" rel="stylesheet" type="text/css">
+    <style> body{background: url("image/pic.png");} </style>
+
+  </head>
+
+  <body>
+    <header>
+    <nav>
+    <div class="navbar">
+            <h1 class="logo"> My Logo </h1>
+
+            <ul>
+              <li class="nav-items"><a href="#">Portfolio</a></li>
+              <li class="nav-items"><a href="#">About</a></li>
+              <li class="nav-items"><a href="#">Contact</a></li>
+            </ul>
+            <div class="hamburger">
+                <span class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>
+            </div>
+          </div>
+          </nav>
+        </header>
+
+          <section class="homepage">
+            <div class="intro">
+
+            <h1>I'm Tasnimul <br>Glad to meet you!</h1>
+            <br>
+            <br>
+            <p>
+              I am a software developer. I have done some interesting projects using
+              react js, node js and ruby on rails. I am learning new things and
+              implementing them instantly. I can help you build a project, feature
+              or website look through some of work and experience if you like what
+              you see and have a project you need coded, don't hesitate to contact
+              me.
+            </p>
+
+            <h2 class="socials"> Let's Connect</h2>
+                       <ul class="icon">
+                           <li><a href="#"><img src="image/angellist.svg" alt="" /></a></li>
+                           <li><a href="#"><img src="Image/github.svg" alt="" /></a></li>
+                           <li><a href="#"><img src="Image/medium.svg" alt="" /></a></li>
+                           <li><a href="#"><img src="Image/twitter.svg" alt="" /></a></li>
+                       </ul>
+                   </div>
+               </section>
+
+
+
+               <script src="./app.js"></script>
+
+
+
+
+
+
+
+
+
+
+
+</body>
```

Comment:
- [ ] You did well working on this project. But so far, I will suggest you always try to stick with the Figma design and specifications. Currently, your design differs a bit from the Figma design.

Your Design | Figma Design |
:---------------:|:-------------------:|
![c1](https://user-images.githubusercontent.com/39523312/148653962-9f3aca53-0e5c-4bc8-a292-42670d64ece0.png) |![c2](https://user-images.githubusercontent.com/39523312/148653965-cb21186f-718f-4d95-b774-cb31e7b94ab9.png)